src/core/CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
 set(_srcs
-    charpak_mod.F90
+    precision_mod.F90
```

Comment:
it was already in the list. better to keep alphabetical order to more easily avoid this